server/svix-server/src/core/security.rs

```diff
@@ -60,6 +61,9 @@ impl Permissions {
 pub struct CustomClaim {
     #[serde(rename = "org", default, skip_serializing_if = "Option::is_none")]
     pub organization: Option<String>,
+
+    #[serde(rename = "evt_ff", default, skip_serializing_if = "Option::is_none")]
+    pub event_feature_flags: Option<EventTypeFeatureFlagSet>,
```

Comment:
Does this need to be an `Option`? Could we get away with an empty set as the Default?

Reply:
Yes, empty set as default works, I only made it an `Option` because I'd expect this will not be used for most requests, therefore:
1. Keeps the JWT smaller. Not a big deal but would save an `"evt_ff":[]`.
2. Performance. If it's not an option then a `HashSet` will be allocated even when not used. But that might very well be premature optimization.

Will change to non-`Option`al 👍

Reply:
@svix-andor, not serializing an empty evt_ff in tokens is a worthy goal. I dunno whether I'd want to sacrifice it.

Reply:
I think empty collections usually don't allocate, but point 1 (keeping the JWT smaller) seems reasonable.

I'm ok if you leave as is.

Reply:
What I said was in addition to your comment. As in, change `skip_serializing_if` to check for HashMap::is_empty or whatever.

Reply:
Done as discussed:
* No longer an option, just a direct `FeatureFlagSet`
* Added serde `default` annotation
* Changed to `skip_serializing_if = "FeatureFlagSet::is_empty"`